src/start/CompatibilityScreen.js

```diff
@@ -15,17 +16,25 @@ const styles = StyleSheet.create({
     color: 'white',
     fontSize: 20,
     margin: 8,
-  }
+  },
 });
 
 export default class CompatibilityScreen extends React.Component {
-
+  handleClick = (link: string) => {
+    Linking.canOpenURL(link).then(supported => {
+      if (!supported) {
+        Alert.alert('Loading error', 'Could not open the store.');
+      } else {
+        Linking.openURL(link);
+      }
+    }).catch(err => console.log('An error occurred ', err)); // eslint-disable-line
+  }
   render() {
     return (
       <View style={styles.screen}>
         <Logo />
-        <Text style={styles.text}>This app is too old!</Text>
-        <Text style={styles.text}>Please download the latest version from the App Store.</Text>
+        <Text style={styles.text}>This is an outdated version of Zulip.</Text>
```

Comment:
Nice improvement! I'd keep the 'Please download the latest version from the App Store:' text though so it's more of a call to action